ts/components/dialog/user-settings/pages/ConversationSettingsPage.tsx

```diff
@@ -96,7 +95,7 @@ export function ConversationSettingsPage(modalState: UserSettingsModalState) {
           baseDataTestId="audio-message-autoplay"
           active={audioAutoPlay}
           onClick={async () => {
-            dispatch(toggleAudioAutoplay());
+            await window.setSettingValue(SettingsKey.audioAutoplay, !audioAutoPlay);
```

Comment:
should we (almost)  always use `setSettingValue` instead of Storage.put?

Reply:
maybe, ive cleaned this up a lot in my settings pr, making more unified setting setters